crates/sui-node/src/lib.rs

```diff
@@ -897,10 +897,12 @@ impl SuiNode {
                 .get_sui_system_state_object_during_reconfig()
                 .expect("Read Sui System State object cannot fail");
             if let Err(err) = self.end_of_epoch_channel.send(latest_system_state.clone()) {
```

Comment:
Why not only read and send the latest_system_state when it is fullnode?

Reply:
Well, because we subscribe to this channel on validators in tests :(